### PR TITLE
Implement breadcrumbs

### PR DESCRIPTION
**Pull request by poly2it**
Implements breadcrumbs from https://github.com/wroyca/libadwaita-vscode-theme/issues/10.

### CODE REVIEW

File: src/vs/workbench/browser/parts/editor/media/tabstitlecontrol.css [hunk 1]
```diff
@@ -411,16 +411,26 @@
...
+}
+
 .monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-breadcrumbs .breadcrumbs-control .outline-element-icon {
 	padding-right: 3px;
 	height: 22px; /* tweak the icon size of the editor labels when icons are enabled */
 	line-height: 22px;
 }
 
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-breadcrumbs .monaco-breadcrumbs {
+	min-height: 28px; /* this overrides the element style height-property */
```
Comment (wroyca) [inline src/vs/workbench/browser/parts/editor/media/tabstitlecontrol.css:425]: In order to adhere to the GNOME guidelines, let's make the breadcrumb background rounded and decrease its width. We should aim for a close-to-perfect alignment of around 902 pixels, although I haven't measured it precisely

```suggestion
	min-height: 28px; /* this overrides the element style height-property */
	border-radius: 6px;
	width: 902px !important;
```

File: src/miscellaneous.css [hunk 2]
```diff
@@ -110,3 +110,18 @@ div[class^="cdr"]
...
+	margin-left: 14px;
+	margin-right: 14px;
+	min-height: 36px;
+}
+
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-breadcrumbs .monaco-breadcrumbs .monaco-breadcrumb-item .codicon-breadcrumb-separator:before {
+	content: '/' !important;
+    font-size: 13px;
+    line-height: 36px;
+    font-family: system-ui, Ubuntu, Droid Sans, sans-serif;
```
Comment (wroyca) [inline src/miscellaneous.css]: Indentation

```suggestion
    content: '/' !important;
    font-size: 13px;
    line-height: 36px;
    font-family: system-ui, Ubuntu, Droid Sans, sans-serif;
```

**Review by wroyca — CHANGES_REQUESTED** [review]
Looks good overall, just left a few suggestions before merging, but once they're taken care of, it should be good to go. Thanks!

**Comment by poly2it** [comment]
Changes look good.

**Pull request merged**